src/test/java/uk/nhs/digital/iucds/middleware/MiddlewareApplicationTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2012-2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package uk.nhs.digital.iucds.middleware;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.util.MimeTypeUtils;
+import com.amazonaws.services.simplesystemsmanagement.AWSSimpleSystemsManagement;
+import microsoft.exchange.webservices.data.core.ExchangeService;
+import microsoft.exchange.webservices.data.core.enumeration.misc.ExchangeVersion;
+import microsoft.exchange.webservices.data.core.enumeration.property.WellKnownFolderName;
+import microsoft.exchange.webservices.data.core.service.item.EmailMessage;
+import microsoft.exchange.webservices.data.core.service.item.Item;
+import microsoft.exchange.webservices.data.property.complex.Attachment;
+import microsoft.exchange.webservices.data.property.complex.AttachmentCollection;
+import microsoft.exchange.webservices.data.property.complex.FileAttachment;
+import microsoft.exchange.webservices.data.search.FindItemsResults;
+import microsoft.exchange.webservices.data.search.ItemView;
+import microsoft.exchange.webservices.data.search.filter.SearchFilter.SearchFilterCollection;
+
+@SpringBootTest
+public class MiddlewareApplicationTest {
+
+  private static final String HTML_FILE = "src/test/resources/output.html";
+  
+  @MockBean
+  private MiddlewareSchedulerTask tasks;
+
+  @Mock
+  private ExchangeService service;
+
+  @Mock
+  private AWSSimpleSystemsManagement ssm;
+
+  @Mock
+  private EmailMessage message;
+
+  @Mock
+  private FindItemsResults<Item> items = new FindItemsResults<Item>();
+
+  @Spy
+  private ArrayList<Item> item = new ArrayList<Item>();
+
+  @Spy
+  private AttachmentCollection attachments;
+
+  @Spy
+  private List<Attachment> attachmectItems;
+
+  @Mock
+  private FileAttachment attachment;
+
+  private MiddlewareSchedulerTask getSut() throws Exception {
+    return new MiddlewareSchedulerTask(service);
+  }
+
+  @Test
+  public void contextLoads() throws Exception {
```

Comment:
This test is still failing consistently. If it requires some parameters to run properly that we cannot add into the source, this should not be part of the build as is. We need to update this test so it run and passes regardless of the environment it is run.